Client/Application/LeanplumIntegration.swift

```diff
@@ -185,7 +198,14 @@ class LeanPlumClient {
             log.error("LeanplumIntegration - Could not be started")
             return
         }
-
+        
+        // Adding a self generated user device id because on firefox beta
+        // device id always remains the same for iPhone
+        if userDeviceId == nil && UIDevice.current.userInterfaceIdiom == .phone {
```

Comment:
I have tested this on my device and the device id remains consistent on iPhone however on iPad it changes on every install. So, talking with LP engineers like they suggestion to add a new user id I am also creating a new device id.